libs/feature/catalog/src/lib/organisations/service/organisations-from-metadata.service.ts

```diff
@@ -163,19 +165,25 @@ export class OrganisationsFromMetadataService
     })
   }
 
-  private mapContactFromOrganisation(
-    organisation: Organisation,
-    contact: MetadataContact
-  ): MetadataContact {
-    const logoUrl = organisation.logoUrl
-      ? getAsUrl(`${organisation.logoUrl}`)
-      : contact.logoUrl
-    return {
+  private hydrateResourceContactWithOrg(
+    metadataRecord: MetadataRecord,
+    organisation: Organisation
+  ): MetadataRecord {
+    const firstResourceContact = metadataRecord.resourceContacts[0]
+    const logoUrl =
+      firstResourceContact.logoUrl ||
+      (organisation.logoUrl ? getAsUrl(`${organisation.logoUrl}`) : null)
+    const mappedOrg = {
```

Comment:
I think that we said that the organisation props would only hydrate missing contact props, didn't we ? :thinking: 
Would mean that the priority is the contact email, the contact logo, the contact name etc...